### ISSUE TITLE
Unable to sideload Spotube-iOS.ipa latest Nightly Build via SideStore/AltStore

### ISSUE THREAD

**Issue by Sakhalinfox** [KRTirtho/spotube#1160]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi,

I've been attempting to sideload the latest 'Spotube-iOS.ipa' nightly build using SideStore/Altstore running on iOS version 17.3, but the build fails to install.

This is the error I see in Sidestore during the install which fails:
Error code: com.rileytestut.AltSign error 0

Error message: /Users/runner/Library/Developer/Xcode/DerivedData/AltStore-dunvejkuqerofjghtqolpzgexuyv/SourcePackages/checkouts/AltSign/Sources/ldid/../ldid-core/ldid.cpp(1368): _assert(): end >= size - 0x10

Anyone else faced this and have a work around or know the cause?

### Expected Behavior

App should install via SideStore/AltStore and run.

### Steps to reproduce

1. Download Spotube-iOS.ipa from https://github.com/KRTirtho/spotube/releases/tag/nightly on iPhone 14 using Safari Browser and save to Files.
2. Click '+' button on SideStore or AltStore and select Spotube-iOS.ipa file
3. Loader to install reaches 3/4th way through and shows "An unknown error occured" 
4. Go to settings on SideStore/AltStore and view Error log. Displays above mentioned error

### Operating System

iOS 17.3

### Spotube version

Spotube-iOS.ipa Nightly build 24/01/2024

### Installation source

GitHub Actions (Nightly Binary)

### Additional information

![image](https://github.com/KRTirtho/spotube/assets/121224529/1d63cb6c-55b1-4c90-8810-613ccb8d58ab)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by SrIzan10** [comment]
that nightly version is from a year ago, try latest

**Comment by SrIzan10** [comment]
Update: tried latest tag's ipa and this still happens, but I've read somewhere that using Sideloadly works.

**Comment by Sakhalinfox** [comment]
> Update: tried latest tag's ipa and this still happens, but I've read somewhere that using Sideloadly works.

Yes, the issue persists even with the latest released IPA v3.4.1 (https://github.com/KRTirtho/spotube/releases/download/v3.4.1/Spotube-iOS.ipa). I was able to install it with Sideloadly, but I think it's a bit easier to refresh AppID every 7 days via SideStore via the Wireguard-VPN/Wirelessly.

**Comment by Guitou031** [comment]
Hello I reproduced the error I confirm

**Comment by EnTaroAiur** [comment]
This issue is still occurring with me.

**Comment by Kairixir** [comment]
The issue happens to me too. Seems like an issue with the sideloader rather than with Spotube.

https://github.com/SideStore/SideStore/issues/227#issuecomment-1428168216


If it happens to you too, go to this issue and raise awareness for the AltStore community:
https://github.com/altstoreio/AltStore/issues/1034#issue-1295526593

**Comment by arjpar** [comment]
@Kairixir it seems to be a problem with Spotube and how it's being built. See [here](https://github.com/altstoreio/AltStore/issues/1034#issuecomment-1998081907).
